controller.go

```diff
@@ -72,24 +88,19 @@ func (c *controller) Run(ctx context.Context) error {
 				err := stream.Subscribe(streamCtx, MeteredEventHandler(c.Observability.Meter, n, EventHandlerFunc(c.enqueue)))
 				if err != nil {
 					c.Error("Failed subscribing to stream", "error", err)
```

Comment:
Maybe worth adding the streamId to the log message.

Reply:
👍